### PR TITLE
Allow Using External Players in Settings and Player

### PR DESCRIPTION
**Pull request by jaruba**
Untested, awaiting `stremio-core` changes, based on the discussion had today with @unclekingpin and @elpiel

### CODE REVIEW

File: src/routes/Settings/useProfileSettingsInputs.js [hunk 1]
```diff
@@ -213,15 +213,15 @@ const useProfileSettingsInputs = (profile) => {
...
+        checked: profile.settings.playerType === 'external',
         onClick: () => {
             core.transport.dispatch({
                 action: 'Ctx',
                 args: {
                     action: 'UpdateSettings',
                     args: {
                         ...profile.settings,
-                        playInExternalPlayer: !profile.settings.playInExternalPlayer
+                        playerType: profile.settings.playerType !== 'external' ? 'external' : 'internal'
```
Comment (sleeyax) [inline src/routes/Settings/useProfileSettingsInputs.js]: I don't have the whole context here, but possible typo?

```suggestion
                        playerType: profile.settings.playerType === 'external' ? 'external' : 'internal'
```

Comment (sleeyax) [reply src/routes/Settings/useProfileSettingsInputs.js]: never mind I commented too soon. this is probably supposed to be flipped.

Comment (jaruba) [reply src/routes/Settings/useProfileSettingsInputs.js]: it needs to update the value to the opposite of the current value (like a toggle), so the code is already correct, i think

**Comment by jaruba** [comment]
Tested on:

[MacOS] Disabled, VLC (with and without server attached, all is working as expected)
[iPad] Disabled, VLC, Outplayer, Infuse (Infuse is the only one that is acting wonky and refuses to play some streams, but works with others, we may consider disabling support for it)
[Android] Disabled, Allow Choosing, VLC, Just Player, MX Player (all are working as expected)

**Review by elpiel — APPROVED** [review]

**Comment by elpiel** [comment]
LGTM!

**Comment by elpiel** [comment]
Stremio-web settings were tested:

- Andoird (firefox) Has options: Allow choosing, VLC Just player MX Player, tested with: Allow choosing, VLC
- Ubuntu 20.04 (firefox), has options: VLC , tested with: VLC

With both cases covered:
- With a running server.js
- Without a running server.js

**Review by TRtomasz — APPROVED** [review]
Tested on Android
 and Windows
Works fine, except one thing that has been opened https://github.com/Stremio/stremio-web/issues/378 but not a biggy so can be done after initial release

**Pull request merged**